packages/botonic-plugin-nlu/src/model-handler.ts

```diff
@@ -71,9 +71,9 @@ export class ModelHandler {
     const lang = detectLang(input, this.languages)
     const { model, modelData } = this.modelInfo[lang]
     const tensor = inputToTensor(input, this.modelInfo[lang].preprocessor)
-    const prediction = model.predict(tensor)
+    const prediction = model.predict(tensor as any)
```

Comment:
@vanbasten17 
we don't like casts to any unless necessary (that's why we use typescript)
btw, why was this merged without review process?

Reply:
this was the suggestion coming from tfjs types. If this was done intentionally, the suggestion said that you had to cast it to any/unknown type. For the rest I have merged it because it bumps the same as dependabot suggested.